src/dev.js

```diff
@@ -60,6 +60,10 @@ const wdm = webpackDevMiddleware(compiler, {
   },
 })
 
+wdm.waitUntilValid(() => {
+  console.log(`\n[Force] Booted on ${process.env.APP_URL}.\n`)
+})
```

Comment:
Noticed this was missing post webpack-5 upgrade due to no more friendly errors plugin

Reply:
This plugin not being here is def a big hit on the DX as we no longer get nice in-browser overlays pointing at syntax errors, and other clear console output. Will explore in another PR as I think react-fast-refresh comes with something packaged in; otherwise, there's likely a webpack 5 fork of friendly errors

Reply:
Could you elaborate on what you mean here `this was missing post webpack-5`. The Webpack 5 upgrade did not add nor remove any plugins.